app/backend/approaches/chatreadretrieveread.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Coroutine, List, Literal, Optional, Union, overload
+from typing import Any, Coroutine, Dict, List, Literal, Optional, Union, overload
```

Comment:
Dict en List hoeven niet meer geimporteerd te worden. Deze moeten worden vervangen door dict en list

Reply:
This is currently everywhere in the code, for now I won't change it but we could do so at a later stage!